MarsMission/Controllers/DetailsViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DetailsViewController.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/03.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource{
```

Comment:
Spacing between bracket and UICollectionViewDataSource

Reply:
<img width="757" alt="Screenshot 2020-11-05 at 06 23 51" src="https://user-images.githubusercontent.com/55865505/98197567-b70df000-1f2f-11eb-90e6-b19954438692.png">